client/state/current-user/email-verification/reducer.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { createReducer } from 'state/utils';
+// import { combineReducers } from 'redux';
```

Comment:
Uncommented this line - it's needed as of 406d3b94d685804a5166b7b394273e0a74bb60f9